src/vfcwrapper/funcinstr.c

```diff
@@ -0,0 +1,261 @@
+/************************************************************
+*                       Hash Table                          *
+************************************************************/
+
+#define table_size 100
```

Comment:
the wrapper already uses a hash implementation for addresses in delta-debug. Can we reuse part of the implementation, instead of having two different implementations ?